joomla/joomla-cms
# Remove never needed nor used JED Update Server from new installs

### Summary of Changes

Following Michaels tip: https://github.com/joomla/joomla-cms/pull/13041#issuecomment-263612114

> we should just delete that never used JED channel already) 

Remove never needed nor used JED Update Server from new installs

### Testing Instructions

- Install staging.
- go to the update sites view
- check theat the Joomla! Extension Directory entry is there
- install this branch: https://github.com/zero-24/joomla-cms/archive/remove_jedUpdateserv.zip
- check that this update server is not there.

### Documentation Changes Required

None as it was never used.

see: https://github.com/joomla/update.joomla.org/pull/31
